docs/en/authoring.md

```diff
@@ -69,7 +67,7 @@ lang: sv
 Normalt markdown-innehåll följer...
 ```
 
-{% include info title="Common Fields" content="Note how the permalink needs to be the same for both pages and how we have added lang: sv in order to specify that this is the swedish version." %}
+{% include info title="Common Fields" content="Note how the pagename needs to be the same for both pages and how we have added lang: sv in order to specify that this is the swedish version." %}
```

Comment:
minor: worth adding a section to the docs about restructuring the TOC and renaming pages?

Reply:
Added an info include discussing this to the TOC page.